### PR TITLE
Add Support for saving the current weekday in URL

### LINKED ISSUE #47: The selected date should be part of the url
The selected date should be part of the url, otherwise the date will always be reset to the current date on reload.  
It would also be way easier to share links of the selected date.

### PR DESCRIPTION
**Pull request by GamerBene19**
If there is a weekday parameter it will take priority over the previous logic (choosing today, or next monday if today lies on the weekend). 

This PR will close #47 and #46

### CODE REVIEW

File: src/components/navigation/weekday-selection/WeekdaySelection.tsx [hunk 1]
```diff
@@ -10,12 +11,32 @@ import styles from './WeekdaySelection.module.scss';
...
+    if (!router.isReady) return;
+
+    // Get and parse the weekday from URL
+    const { weekday: weekdayURL } = router.query;
+    const weekdayURLParsed =
+      typeof weekdayURL === 'string' ? parseInt(weekdayURL) : -1;
+
+    const nextOpenWeekday = afterFriday ? 0 : currentWeekday;
+    const weekdayToSet =
+      weekdayURLParsed >= 0 ? weekdayURLParsed : nextOpenWeekday;
```
Comment (stormofice) [inline src/components/navigation/weekday-selection/WeekdaySelection.tsx]: I strongly prefer pascal case names for acronyms, so this (and similar variables) would become:
`weekdayUrlParsed`
This is also recommended by:
- [Google](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined)
- [Microsoft](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN)

(Similar names are used across this file, but I only included this suggestion here)

File: src/components/today-overview/TodayOverview.tsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ const TodayOverview = () => {
         .add(navData ? navData.selectedWeekday : 0, 'day')
         .format('YYYY-MM-DD'),
     },
+    skip: navData && navData.selectedWeekday < 0,
```
Comment (stormofice) [inline src/components/today-overview/TodayOverview.tsx:32]: grafik

**Review by stormofice — COMMENTED** [review]
Looking good, just got some questions 😄

File: src/components/today-overview/TodayOverview.tsx (hunk 2 above)
Comment (GamerBene19) [reply src/components/today-overview/TodayOverview.tsx:32]: 10/10 Meme. Erstell ein issue dafür. Das ist imo seperat von diesem branch anzusehen

Comment (stormofice) [reply src/components/today-overview/TodayOverview.tsx:32]: Sure, will do

File: src/components/navigation/weekday-selection/WeekdaySelection.tsx (hunk 1 above)
Comment (GamerBene19) [reply src/components/navigation/weekday-selection/WeekdaySelection.tsx]: Done.

Comment (stormofice) [reply src/components/navigation/weekday-selection/WeekdaySelection.tsx]: Could this also be changed [here](https://github.com/mensatt/frontend/pull/71/files#diff-3b0382bc231dce3a6e9fe1768b0b63933221ce3654fd7b5d60bbdd5423dd18c4R23)?

Comment (GamerBene19) [reply src/components/navigation/weekday-selection/WeekdaySelection.tsx]: Yes. Change will be pushed shortly

Comment (GamerBene19) [reply src/components/navigation/weekday-selection/WeekdaySelection.tsx]: Done.

**Review by stormofice — DISMISSED** [review]

**Review by stormofice — APPROVED** [review]

**Pull request merged**